orc8r/gateway/configs/templates/td-agent-bit.conf.template

```diff
@@ -49,7 +49,7 @@
     Name     forward
     Tag      docker.log
     Listen   0.0.0.0
-    port     24224
+    port     24221
```

Comment:
Are FeG dockers still able to export docker service logs if we change this value?

Reply:
you mean show `docker-compose log`?  If that is what you meant, then yes.

I was not able to find any reference to port `24224` that was not related to configuration

Reply:
I was wondering if docker logs still get forwarded to fluent propoerly if we change this.

Does this input logic mean that it tags any input received from port 24221 as `docker.log`? If it is, I was wondering if there was any other configuration that specifies 24221 as the forwarding port to stream logs coming out of docker containers. 

From a quick glance in the repo I didn't seem to find anything, but just making sure :)